docs/components/sidecar.md

```diff
@@ -55,7 +55,7 @@ $ prometheus \
   --web.enable-lifecycle
 ```
 
-```bash
+```bashas
```

Comment:
I don't know if it is intentional or just a mistake, I don't think `bashas` is a thing.
```suggestion
```bash
```